src/types/complex-types/object.ts

```diff
@@ -98,7 +98,15 @@ export class ObjectType<S, T> extends ComplexType<S, T> implements IModelType<S,
             // todo assert plain object
             if (actions && isPlainObject(actions)) {
                 Object.keys(actions).forEach(name => {
-                    addHiddenFinalProp(self, name, createActionInvoker(self, name, actions[name]))
+                    const action = actions[name]
+                    if ((action as any).isAsyncMSTAction === true)
```

Comment:
Uhm, what about having a modifier like "silent(...action...)" instead of specifically tie it to async actions?